src/coreclr/jit/valuenum.cpp

```diff
@@ -6770,6 +6770,138 @@ const char* ValueNumStore::VNRelationString(VN_RELATION_KIND vrk)
 }
 #endif
 
+bool ValueNumStore::IsVNPhiDef(ValueNum vn)
```

Comment:
This looks unused

Reply:
Yeah, I'll remove it soon, if I don't find some other use for it.